node.gyp

```diff
@@ -929,6 +928,14 @@
             'deps/uv/uv.gyp:libuv'
           ]
         }],
+        [ 'node_shared_nghttp2=="false"', {
+          'dependencies': [
+            'deps/nghttp2/nghttp2.gyp:nghttp2'
+          ],
+          'include_dirs': [
+            'deps/nghttp2/lib/includes'
+          ]
```

Comment:
The `include_dirs` is probably not necessary, nghttp2.gyp also exports it through `direct_dependent_settings`.